README.md

```diff
@@ -66,11 +66,14 @@ This plugin provides different template modules for different kinds of projects.
     * [maven-failsafe-plugin](https://maven.apache.org/surefire/maven-failsafe-plugin/)
     * jacoco coverage configuration for integration tests
 #### `maven_central`
-*Not yet implemented.*
 
-This module will check required configuration for releasing on maven central in the future.
-You can, however, already add this module. By that you can't forget to add it later.
+This module checks the required configuration for releasing on maven central.
 
+* required maven plugins
+    * [maven-gpg-plugin](https://maven.apache.org/plugins/maven-gpg-plugin/)
+    * [maven-deploy-plugin](https://maven.apache.org/plugins/maven-deploy-plugin/)
+    * [nexus-staging-maven-plugin](https://github.com/sonatype/nexus-maven-plugins/tree/master/staging/maven-plugin)
```

Comment:
I think we're missing some other plugins here. I'm not sure if you check them in some other module, but the full list is: 
"nexus-staging-maven-plugin", "maven-source-plugin", "maven-gpg-plugin", "maven-javadoc-plugin", "maven-deploy-plugin".
I've also added a template with them to the release robot: https://github.com/exasol/release-robot/blob/feature/50_add_validations_for_maven_release/doc/user_guide/maven_central_release_pom_file_template.md

Reply:
Yes. This is just the plugins that were mentioned in the issue. The other plugins I will add when fixing issue #18

Reply:
Okay